src/osd/PG.cc

```diff
@@ -3994,6 +4009,16 @@ void PG::chunky_scrub(ThreadPool::TPHandle &handle)
 	  scrubber.reserved_peers.clear();
 	}
 
+	{
+	  hobject_t oid = Scrub::make_scrub_object(info.pgid.pgid);
```

Comment:
There might be two shards on the same OSD for the same PG.  The scrub object should probably be built using the spg_t rather than the pg_t.